core/src/test/scala/unit/kafka/log/remote/RemoteIndexCacheTest.scala

```diff
@@ -321,16 +322,16 @@ class RemoteIndexCacheTest {
     // Give the thread cleaner thread some time to throw an exception
     Thread.sleep(100)
     // Verify that Cleaner thread is still running even when exception is thrown in doWork()
-    var threads = TestUtils.numThreadsRunning(REMOTE_LOG_INDEX_CACHE_CLEANER_THREAD, isDaemon = true)
+    var threads = getRunningCleanerThread
     assertEquals(1, threads.size,
-      s"Found unexpected ${threads.size} threads=${threads.map(t => t.getName).mkString(", ")}")
+      s"Found unexpected ${threads.size} threads=${threads.stream().map(t => t.getName).collect(Collectors.joining(", "))}")
```

Comment:
`threads.stream().map[String](t => t.getName).collect(Collectors.joining(", "))`

You can define the `type` explicitly to fix the type mismatch in scala 2.12

Reply:
fix it, and run `./gradlew clean build -x test -PscalaVersion=2.12` is build successful

Reply:
nice job!